.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
```

Comment:
Why not use headings:
```suggestion
#### Describe the bug
```

Reply:
** surrounding content and followed by newline is rendered as a heading in issue templates. (doesn't look like it if you directly view the template in markdown in file view, but it renders different when actually creating an issue.)